src/Service/Modules/PurchaseOrderModule.cs

```diff
@@ -62,18 +66,39 @@ private readonly
             this.Get("/purchasing/purchase-orders/tariffs", this.SearchTariffs);
             this.Get("/purchasing/purchase-orders/{OrderNumber:int}/over-book", this.SearchPurchaseOrders);
             this.Put("/purchasing/purchase-orders/{OrderNumber:int}/over-book", this.AllowOverbook);
+
+            this.Get("/purchasing/purchase-orders/reqs", this.SearchReqs);
+            this.Get("/purchasing/purchase-orders/reqs/{id:int}", this.GetReq);
+            this.Put("/purchasing/purchase-orders/reqs/{id:int}", this.UpdateReq);
+            this.Post("/purchasing/purchase-orders/reqs", this.CreateReq);
         }
 
-        private async Task GetCurrencies(HttpRequest req, HttpResponse res)
+        private async Task GetApp(HttpRequest req, HttpResponse res)
         {
-            var result = this.currencyService.GetAll();
+            await res.Negotiate(new ViewResponse { ViewName = "Index.html" });
+        }
+
+        private async Task AllowOverbook(HttpRequest req, HttpResponse res)
+        {
+            var resource = await req.Bind<PurchaseOrderResource>();
+            resource.Privileges = req.HttpContext.GetPrivileges();
```

Comment:
no need to attach the privileges to the resource here. Just pass em straight to the facadeservice

Reply:
This is part of Ross's overbook work, I think I'll let him merge his pr into main first and then pull main into this branch again - I'm pretty sure it'll be changed on his branch